src/module_wrap.cc

```diff
@@ -602,9 +602,20 @@ static MaybeLocal<Promise> ImportModuleDynamically(
     UNREACHABLE();
   }
 
+  Local<Object> assertions =
+    Object::New(isolate, v8::Null(env->isolate()), nullptr, nullptr, 0);
+  for (int i = 0; i < import_assertions->Length(); i += 2) {
+    assertions
+        ->Set(env->context(),
+              Local<String>::Cast(import_assertions->Get(env->context(), i)),
+              Local<Value>::Cast(import_assertions->Get(env->context(), i + 1)))
```

Comment:
Can you use `.As<String/Value>()` ? I don't think we use `Cast` anywhere in Node.js.

Reply:
I used for inspiration this snippet: https://github.com/nodejs/node/blob/fa7cdd6fc9dffa8139f9350f54959b01bf7a1151/src/module_wrap.cc#L292-L298

There are other occurences of `Cast` in the codebase (18 results in 7 files according to my text editor), would it be OK to address those in a separate PR?

Reply:
Ok, sure.

Reply:
PR to use `As()` instead of `Cast()` in all places: https://github.com/nodejs/node/pull/40287 :)